validation/src/shared_table/mod.rs

```diff
@@ -395,7 +395,7 @@ impl SharedTable {
 	}
 
 	/// Get the parent hash this table should hold statements localized to.
-	pub fn consensus_parent_hash(&self) -> &Hash {
+	pub fn session_parent_hash(&self) -> &Hash {
```

Comment:
I think this should stay as is (because it is a parent hash of a block in the consensus)